spec/models/version_metadata_spec.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe VersionMetadata do
+  let(:version_metadata) { FactoryBot.create(:version_metadata) }
+  let(:druid) { version_metadata.druid }
+  let(:version) { version_metadata.version }
```

Comment:
do the `druid` and `version` vars get used at all?  cruft?

Reply:
Good catch, it is cruft.  Removing.